src/model/ModelResources.i

```diff
@@ -98,6 +98,8 @@ MODELOBJECT_TEMPLATES(DefaultScheduleSet);
 MODELOBJECT_TEMPLATES(SpectralDataField); // Helper class defined in MaterialPropertyGlazingSpectralData
 MODELOBJECT_TEMPLATES(MaterialPropertyGlazingSpectralData);
 MODELOBJECT_TEMPLATES(MaterialPropertyMoisturePenetrationDepthSettings);
+MODELOBJECT_TEMPLATES(MaterialPropertyPhaseChange);
```

Comment:
You also need to SWIG to TemperatureEnthalpy helper. this is what your build error is on windows. I'll fix it in 2150199deb7fcf165e2ad695d8d867344dc1f980